### PR TITLE
Add architecture diagram

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/45878

### CODE REVIEW

File: docs/GettingStarted.asciidoc [hunk 1]
```diff
@@ -59,6 +59,10 @@ application takes care of distributing test jobs among workers. Web
 application and workers don't have to run on the same machine but
 can be connected via network instead.
 
+Note that the diagram shown above is simplified. There exists
+https://raw.githubusercontent.com/os-autoinst/openQA/master/docs/images/architecture.svg[a more sophisticated version]
```
[the remark this answers is not in the document]
Comment (Martchus) [reply docs/GettingStarted.asciidoc]: ok, I'll try tomorrow

**Comment by aaannz** [comment]
You have 7 typos wrt service names e.g. - openq-gru instead of openqa-gru.

**Comment by Martchus** [comment]
@aaannz Search & replace applied :-)

File: docs/GettingStarted.asciidoc [hunk 2]
```diff
@@ -59,6 +59,11 @@ application takes care of distributing test jobs among workers. Web
 application and workers don't have to run on the same machine but
 can be connected via network instead.
 
+Note that the diagram shown above is simplified. There exists
+link:images/architecture.svg[a more sophisticated version]
```
[the remark this answers is not in the document]
Comment (Martchus) [reply docs/GettingStarted.asciidoc]: I don't understand what you mean? I don't want to include that (huge) diagram directly in the document (so the `link:` is intended). If you want an automated conversion of the GraphML to SVG - I don't think that's possible.

**Comment by Martchus** [comment]
Using relative links, fixed typos and mentioned the underlying source file for editing. The failing tests can be safely ignored for that kind of change.

**Comment by Martchus** [comment]
So is this now ok?

**Comment by Martchus** [comment]
I see now further objections.

**Pull request merged**